### ISSUE TITLE
gmenv install fails for new installs

### ISSUE THREAD

**Issue by joshua-rutherford** [greymatter-io/homebrew-greymatter#6]
Both @dborncamp and I were unable to run `gmenv install` on a clean installation of `gmenv` from homebrew.  The following error was encountered:

```
$ gmenv install 1.4.2                             
Installing Grey Matter v1.4.2
Downloading release tarball from https://nexus.production.deciphernow.com/repository/raw-hosted/greymatter/gm-cli/greymatter-v1.4.2.tar.gz
######################################################################################################################################################################################################################################################### 100.0%
mkdir: /usr/local/Cellar/gmenv/0.2.2/versions/1.4.2: Not a directory
Failed to make directory /usr/local/Cellar/gmenv/0.2.2/versions/1.4.2
```

Running `mkdir -p /usr/local/etc/gmenv/versions` resolves the issue which should be done by a post install operation.  Not sure why this is failing.

**Comment by joshua-rutherford** [comment]
Issue is that the syntax is wrong for the version path:
```
$ cat temporary.rb 
name = 'gmenv'
HOMEBREW_PREFIX = '/usr/local'
versions_path = HOMEBREW_PREFIX/"etc/#{name}/versions"
puts versions_path 
$ ruby temporary.rb 
Traceback (most recent call last):
temporary.rb:3:in `<main>': undefined method `/' for "/usr/local":String (NoMethodError)
```
Should be:
```
$ cat temporary.rb 
name = 'gmenv'
HOMEBREW_PREFIX = '/usr/local'
versions_path = "#{HOMEBREW_PREFIX}/etc/#{name}/versions"
puts versions_path 
$ ruby temporary.rb
/usr/local/etc/gmenv/versions
```
As a result the post install operation is failing and none of that setup is occuring.

**Comment by chrisbsmith** [comment]
Closed by #7
